lago/utils.py

```diff
@@ -421,6 +421,10 @@ def deepcopy(original_obj):
     """
     if isinstance(original_obj, list):
         return list(deepcopy(item) for item in original_obj)
+    elif isinstance(original_obj, collections.OrderedDict):
+        return collections.OrderedDict(
+            (key, deepcopy(val)) for key, val in original_obj.items()
+        )
```

Comment:
@david-caro can give an explanation (which I remember was quite long) on why not. IIRC, it's not really doing only deepcopy?